### ISSUE TITLE
Scroll sync should work from either MD file or Preview pane

### ISSUE THREAD

**Issue by girlandhercode** [neilsustc/vscode-markdown#5]
Excellent extension and thank you for taking the time to build it!  I curious if you could make the scroll sync work whether I'm scrolling over the ``.md`` file OR the preview. Scrolling syncs fine when mouse is over the preview tab, but when I want to scroll thru for edits in the ``.md`` side the preview side remains static. Maybe I have this configured incorrectly or there is a setting? Any advice is appreciated!

**Comment by neilsustc** [comment]
I'm afraid that it's an upstream issue, see [#19459](https://github.com/Microsoft/vscode/issues/19459). 😞

Currently, there are two available settings in vscode,

```json
// Scrolls the markdown preview to reveal the currently selected line from the editor.
"markdown.preview.scrollPreviewWithEditorSelection": true,
// When the markdown preview is scrolled, update the view of the editor.
"markdown.preview.scrollEditorWithPreview": true,
```

As a workaround, when you click somewhere in the `.md` file, the preview will scroll to that point.

**Comment by neilsustc** [comment]
This is implemented with vscode v1.21.0
